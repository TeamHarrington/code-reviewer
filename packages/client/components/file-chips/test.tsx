import { FileChips } from '.'
import renderer from 'react-test-renderer'
import React from 'react'

test('File Chips Component', () => {
  const component = renderer.create(<FileChips files={[]} />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
