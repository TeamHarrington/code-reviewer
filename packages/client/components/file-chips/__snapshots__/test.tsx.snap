// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File Chips Component 1`] = `
<div
  className="file-chips__Container-sc-14bl2q5-0 ebnvxc"
/>
`;
